source/server/http_time_tracking_filter_config.cc

```diff
@@ -0,0 +1,55 @@
+#include <string>
+
+#include "envoy/registry/registry.h"
+
+#include "external/envoy/source/common/protobuf/message_validator_impl.h"
+
+#include "api/server/response_options.pb.h"
+#include "api/server/response_options.pb.validate.h"
+
+#include "server/http_time_tracking_filter.h"
+
+namespace Nighthawk {
+namespace Server {
+namespace Configuration {
+
+class HttpTimeTrackingFilterConfig
+    : public Envoy::Server::Configuration::NamedHttpFilterConfigFactory {
+public:
+  Envoy::Http::FilterFactoryCb
+  createFilterFactoryFromProto(const Envoy::Protobuf::Message& proto_config, const std::string&,
+                               Envoy::Server::Configuration::FactoryContext& context) override {
+
+    auto& validation_visitor = Envoy::ProtobufMessage::getStrictValidationVisitor();
```

Comment:
can we be explicit here, instead of auto.  I'm not seeing a place nearby where that type becomes obvious